dbt_platform_helper/providers/aws/exceptions.py

```diff
@@ -12,16 +12,34 @@ def __init__(self, addon_name: str, application_name: str, environment: str):
         )
 
 
+IMAGE_NOT_FOUND_TEMPLATE = """An image labelled "{image_ref}" could not be found in your image repository. Try the `platform-helper codebase build` command first."""
+
+
 class ImageNotFoundException(AWSException):
     def __init__(self, image_ref: str):
+        super().__init__(IMAGE_NOT_FOUND_TEMPLATE.format(image_ref=image_ref))
+
+
+MULTIPLE_IMAGES_FOUND_TEMPLATE = (
+    'Image reference "{image_ref}" matched the following images: {matching_images}'
```

Comment:
```suggestion
    'Image reference "{image_ref}" is matched by the following images: {matching_images}'
```

Reply:
would resolve my "back-to-front" feeling :P

Reply:
Yes. Or maybe if I implement the opposite behaviour as well, it could be:
```suggestion
    'The following images are possible matches for image reference "{image_ref}": {matching_images}'
```

Reply:
Yes, that's good